pkg/operator/osd/storage.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package osd
+
+import (
+	cephosd "github.com/rook/rook/pkg/cephmgr/osd"
+)
+
+type StorageSpec struct {
```

Comment:
could you put all the structs that are exposed in the yaml in a file called spec.go? Then it's more obvious what comes from the yaml